livingdoc-engine/src/main/kotlin/org/livingdoc/engine/reporting/HtmlElements.kt

```diff
@@ -100,9 +105,9 @@ fun HtmlTable.rows(rows: List<RowResult>) {
 
 private fun createFailedPopupLink(renderContext: HtmlRenderContext, result: Result): Element {
     fun createStacktrace(e: Throwable): String {
-        return StringWriter().use {
-            e.printStackTrace(PrintWriter(it))
-            it.toString()
+        return StringWriter().use { stringWriter ->
```

Comment:
`stringWriter` >> `writer` ?

Reply:
the block uses a printwriter -> stringwriter improves readability